node_exporter/default.toml

```diff
@@ -1,2 +1,5 @@
-listening_port = 9100
-
+[web]
+listen_address = ""
```

Comment:
Does this bind to localhost or 0.0.0.0 when it is blank?

Reply:
Here is how to check:

```
hab studio enter
build node_exporter
source results/last_build.env
hab svc load ${pkg_ident}
hab pkg binlink core/busybox-static netstat
netstat -peanut | grep node_exporter
```

Output

```
# netstat -peanut | grep node_exporter
tcp        0      0 :::9100                 :::*                    LISTEN      1180/node_exporter
```

Since this is an AF_INET6 socket, it handles IPv4, but shows as listening on IPv6 address with ipv4 protocol in netstat.

Confirm connection:

```
# nc -z 0.0.0.0 9100
# echo $?
0
```

Reply:
@bdangit Let me know if you have any other questions!

Reply:
👍 thanks